app/views/posts/index.html.erb

```diff
@@ -0,0 +1,51 @@
+<div class="user-navigation">
+  <a href="<%= user_path(@user) %>">
+    <button class="back-btn">⬅</button>
+  </a>
+</div>
+<section class="user-profile-page">
+  <div class='post-user-info'>
+    <% if @user.photo.present? %>
+      <%= image_tag @user.photo, alt: @user.name, class: 'user-thumbnail' %>
+    <% else %>
+      <span class="user-thumbnail placeholder">No Photo</span>
+    <% end %>
+    <p>Name: <%= @user.name %></p>
+    <p>Number of Posts: <%= @user.posts.count %></p>
+  </div>
+  <div class="user-posts">
+    <% if @user.posts.any? %>
+      <ul class="user-posts-info">
+        <% @user.posts.most_recent_comments(5).each do |post| %>
+  <li>
```

Comment:
- [ ] Well done for calling the `#most_recent_comments` method just like the previous reviewer suggested. However, it is not working as expected because all the comments are showing instead of the 5 most recent ones. Please see attached image below and try to fix it 🔧 
![image](https://github.com/mohammadnewton/blog_app/assets/46839548/aa0e4f41-249a-4a85-b545-beb2560779f8)